src/lib.rs

```diff
@@ -1080,6 +1080,14 @@ impl Graphics2D
             vertex_circle_coords_normalized
         );
     }
+
+    /// Sets the current clip to the rectangle specified by the given
+    /// coordinates. Rendering operations have no effect outside of the
+    /// clipping area.
+    pub fn set_clip(&mut self, x: i32, y: i32, width: i32, height: i32)
```

Comment:
It would be good to provide a way to disable clipping as well, i.e. to disable `GL_SCISSOR_TEST`.

For example, maybe this could take an `Option<Rectangle>`. If this is `None` then clipping is disabled, otherwise it's set to the provided rectangle.